h2o-py/tests/testdir_algos/gbm/pyunit_gbm_monotone_quantile.py

```diff
@@ -0,0 +1,27 @@
+import h2o
+from h2o.estimators import H2OGradientBoostingEstimator
+from tests import pyunit_utils
+
+
+def gbm_monotone_quantile_test():
+    data = h2o.import_file("http://s3.amazonaws.com/h2o-public-test-data/smalldata/gbm_test/autoclaims.csv")
+    data = data.drop(['POLICYNO', 'PLCYDATE', 'CLM_FREQ5', 'CLM_FLAG', 'IN_YY'])
+    train, test = data.split_frame([0.8], seed=123)
+    response = "CLM_AMT5"
+
+    gbm_regular = H2OGradientBoostingEstimator(seed=42, distribution="quantile")
+    gbm_regular.train(y=response, training_frame=train, validation_frame=test)
+
+    # train a model with 1 constraint on MVR_PTS
+    gbm_mono = H2OGradientBoostingEstimator(seed=42, distribution="quantile", monotone_constraints={"MVR_PTS": 1})
+    gbm_mono.train(y=response, training_frame=train, validation_frame=test)
+
+    # train a model with -1 constraint on MVR_PTS
+    gbm_adverse = H2OGradientBoostingEstimator(seed=42, distribution="quantile", monotone_constraints={"MVR_PTS": -1})
+    gbm_adverse.train(y=response, training_frame=train, validation_frame=test)
```

Comment:
missing any assertions

Reply:
I have been inspired by another monotone test, where also no assert is. There is a check on the Java side that monotonicity is not broken. But it is not clear, so I reimplemented the test with different data and check the monotonicity also on the Python side.